build.sbt

```diff
@@ -21,7 +21,8 @@ val collections =
       name := "collection-strawman",
       libraryDependencies ++= Seq(
         "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-        "com.novocode" % "junit-interface" % "0.11" % Test
+        "com.novocode" % "junit-interface" % "0.11" % Test,
+        "org.scalatest" %% "scalatest" % "3.0.1" % Test
```

Comment:
Speaking of tests - is it possible to write a property test which takes `mutable.ArrayBuffer` and `mutable.ArrayDeque` and basically does a fuzz test i.e. any sequence of random operations on them to prove that one can be a drop-in replacement for the other.